doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -237,6 +237,15 @@ The `mz_internal_cluster_replicas` table lists the replicas that are created and
 |------------|----------|-------------------------------------------------------------------------------------------------------------|
 | id         | [`text`] | The ID of a cluster replica. Corresponds to [`mz_cluster_replicas.id`](../mz_catalog/#mz_cluster_replicas). |
 
+## `mz_unstable_cluster_replicas`
+
+The `mz_unstable_cluster_replicas` table lists the replicas that were created during managed cluster alter statement that has not yet finished. The configurations of these replica may differ from the cluster's configuration.
+
+<!-- RELATION_SPEC mz_internal.mz_unstable_cluster_replicas -->
+| Field      | Type     | Meaning                                                                                                     |
+|------------|----------|-------------------------------------------------------------------------------------------------------------|
+| id         | [`text`] | The ID of a cluster replica. Corresponds to [`mz_cluster_replicas.id`](../mz_catalog/#mz_cluster_replicas). |
```

Comment:
Ah, I was envisioning that this table would contain two columns: `id text` and `pending_alter: boolean`. So it would have a row for each replica, and the `pending_alter` column would tell you whether that replica was pending alteration or not.

I don't mind this design either (just the IDs of replicas that are pending) but if we go this route we should call the table `mz_pending_cluster_replicas` so its purpose is more clear.